package.json

```diff
@@ -53,7 +53,7 @@
     "@typescript-eslint/parser": "^5.57.1",
     "@vitejs/plugin-react": "^3.1.0",
     "babel-plugin-styled-components": "^2.0.7",
-    "chromedriver": "^111.0.0",
+    "chromedriver": "^120.0.2",
```

Comment:
we can remove it, since newer version of Nightwatch doesn't require `chromedriver`

Reply:
Done